mkdocs_simple_plugin/plugin.py

```diff
@@ -28,6 +29,7 @@ def on_pre_build(self, config):
                                 config['site_dir'],
                                 self.docs_dir]
         self.ignore_hidden = self.config['ignore_hidden']
+        self.include_extensions = self.config['include_extensions']
```

Comment:
Since this plugin is for mkdocs, which works exclusively with markdown files, I think it makes sense to add it here instead of including it as a default in the config.

```suggestion
        self.include_extensions = ['.md'] + self.config['include_extensions']
```